sw/device/lib/drivers/rv_plic/rv_plic.c

```diff
@@ -407,6 +408,10 @@ void plic_reset_handlers_list(void)
     {
       handlers[i] = &handler_irq_dma;
     }
+    else if ( i == EXT_INTR_1)
+    {
+      handlers[i] = &handler_irq_iffifo;
```

Comment:
as above, this should never be done, HEEP is used in other projects without the testbench, so the plic functions should stay sane - move these parts in the application file - for example by exploiting the irq_dummy or something like this

Reply:
Switched to `plic_assign_external_irq_handler` implementation instead.